python/langsmith/client.py

```diff
@@ -5590,6 +5825,9 @@ def _tracing_control_thread_func(client_ref: weakref.ref[Client]) -> None:
     size_limit: int = batch_ingest_config["size_limit"]
     scale_up_nthreads_limit: int = batch_ingest_config["scale_up_nthreads_limit"]
     scale_up_qsize_trigger: int = batch_ingest_config["scale_up_qsize_trigger"]
+    use_multipart = os.getenv("LANGSMITH_FF_MULTIPART") in ["1", "true"]
```

Comment:
Should we cache this? Probably not terribly important since we create the control thread once per client and it's in the background thread, etc.

Reply:
I think fine